io/jvm-native/src/main/scala/fs2/io/file/FilesPlatform.scala

```diff
@@ -291,6 +291,8 @@ private[file] trait FilesCompanionPlatform {
     def isSameFile(path1: Path, path2: Path): F[Boolean] =
       Sync[F].blocking(JFiles.isSameFile(path1.toNioPath, path2.toNioPath))
 
+    val lineSeparator: String = System.lineSeparator()
```

Comment:
Nit, but I'd use a `def` instead of a `val` since `lineSeparator()` is also a method call.